.github/workflows/claude-nl-suite.yml

```
@@ -1,134 +1,351 @@
+          fail_on_parse_error: true
+          
+        
+      - name: Upload artifacts (lean)
+        if: always()
+        uses: actions/upload-artifact@v4
         with:
-          packageMode: true
-          unityVersion: 2022.3.45f1   # set your exact version
-          projectPath: .
-          githubToken: ${{ secrets.GITHUB_TOKEN }}
+            name: claude-nl-suite-artifacts
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)